SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -520,7 +534,14 @@ void CNEMOEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contai
       Lambda_2 = (Mean_ThermalCond+Mean_ThermalCond_ve)/cv;
 
       Lambda   = (Lambda_1 + Lambda_2)*Area*Area/Mean_Density;
-      nodes->AddMax_Lambda_Visc(iPoint, Lambda);
+
+      if (nodes->GetSymmetry(iPoint))
+         if(nodes->GetSymmetry(iPoint) == 1 && nodes->GetSymmetry(jPoint)) nodes->AddMax_Lambda_Visc(iPoint, 4*Lambda);
+         else if (nodes->GetSymmetry(iPoint) == 2 && nodes->GetSymmetry(jPoint) == 1) nodes->AddMax_Lambda_Visc(iPoint, 8*Lambda);
+         else if (nodes->GetSymmetry(iPoint) == 2 && nodes->GetSymmetry(jPoint) == 2) nodes->AddMax_Lambda_Visc(iPoint, 16*Lambda);
+         else nodes->AddMax_Lambda_Visc(iPoint, 2*Lambda);
+      else nodes->AddMax_Lambda_Visc(iPoint, Lambda);
```

Comment:
Maybe you can explain this in the next dev meeting?
This looks so complex that there has to be a simpler way.

Reply:
also, something missing here in this if

if(nodes->GetSymmetry(iPoint) == 1 && **nodes->GetSymmetry(jPoint)**)

Reply:
@pcarruscag Sure, when will the next dev meeting be ?

The idea behind it is to mimic the eigenvalues given if using a "full" domain. Probably there is a simpler way through manipulation of the Normal values, but not sure how it can be implemented.

Reply:
Would doubling at the end be enough?

Wednesdays at 4pm CET https://meet.jit.si/SU2_DevMeeting

Reply:
If the nodes are in a single symmetry plane, yes. A bit more complex when they are in two symmetry planes, but I will look at it.